### ISSUE TITLE
Shadowrun: Dragonfall Directors Cut (Native)

### ISSUE THREAD

**Issue by mothenjoyer69** [ptitSeb/box86-compatibility-list#187]
### Description
Game opens and plays as expected. Zero issues after 20 or so minutes of playing. My hardware was getting 30fps stable at 1080p on the default graphics settings, so this should be playable on lower end devices with reduced detail.

### Tested on
Phytium ARM Mini PC with a Phytium D2000 (8x core @ 2.4ghz), AMD RX 550 2GB, 32GB DDR4 memory running Debian Sid with Gnome 42.4, using Wine 5.13 from TwisterOS, and Box86 compiled from source.

### Status
Working without issue. 

### Screenshots
![image](https://user-images.githubusercontent.com/97616889/186811259-30141fca-aeac-4341-ad80-71b698be820c.png)

### OpenGL
4.6 

### Additional Information
This game appears to just run fine without any tweaks. The rest of the ShadowRun games use the same engine and should also be playable.

**Comment by mothenjoyer69** [comment]
Game already exists in a report (https://github.com/ptitSeb/box86-compatibility-list/issues/77), happy to delete this one if you need!

**Comment by ptitSeb** [comment]
Yeah, maybe just add an entry to other ticket, to confirm it works well on other hardware :)

**Comment by mothenjoyer69** [comment]
> Yeah, maybe just add an entry to other ticket, to confirm it works well on other hardware :)

Will do!
